lib/config/httpRoutes.js

```diff
@@ -23,300 +23,300 @@
 
 module.exports = [
   // GET (idempotent)
-  {verb: 'get', url: '/users/_me', controller: 'auth', action: 'getCurrentUser'},
```

Comment:
I think we should make an exception with this file. The  old properties ordering method was more developer friendly IMHO :slightly_smiling_face: